app/controllers/families_controller.rb

```diff
@@ -0,0 +1,38 @@
+class FamiliesController < ApplicationController
+  before_action :require_login
+
+  def index
+    @family = Member.where(family_id: session[:user_id])
+  end
+
+  def new
+    @new_family = Family.new
+  end
+
+  def create
+  	@new_family = Family.new(family_params)
+  	if @new_family.save
+  		redirect_to :root
```

Comment:
Set your sublime to use two-space indentation rather than tabs. To do so, open a ruby file, then on the menu choose, Sublime Tex, Preferences, Settings - More, Syntax Specific - User. Make the settings read

```
{ 
  "tab_size": 2,
  "translate_tabs_to_spaces": true
}
```